index.html

```diff
@@ -19,6 +19,32 @@ <h2>KEEPERS</h2>
         </div>
     </header>
 
+    <footer>
+        <div class="footer-content">
+            <div class="footer-section socialmedia">
+                <div class="social-icons">
+                <a href="#"><img src="../australia-zoo/images/icons/github.png" alt="github-icon"></a>
+                <a href="#"><img src="../australia-zoo/images/icons/instagram.png" alt="instagram-icon"></a>
+                <a href="#"><img src="../australia-zoo/images/icons/national-georaphic.png" alt="national-georaphic-icon"></a></div>
+                <div class="footer-bottom">
+                    <p>Copyright 2024 - The Zookeepers</p>
+                </div>
```

Comment:
need some indentation, also please make a ticket to decide which links are we going to use and to put the links once we decided!